examples/awx/list_job_templates.go

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright (c) 2018 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// This example shows how to list the job templates.
+//
+// Use the following command to build and run it with all the debug output sent to the standard
+// error output:
+//
+//	go run list_job_templates.go \
+//		-url "https://awx.example.com/api" \
+//		-user "admin" \
+//		-password "..." \
+//		-ca "ca.pem" \
```

Comment:
Could you name this field & and function caFile? it was confusing for me to read `-ca "ca.pem"`. Naming the parameters the same as the connections functions improves the example readability.

Reply:
Done in the next version of the pull request.